site/src/_pages/docs/1_developing-blocks.mdx

```diff
@@ -23,9 +23,9 @@ If you want to know when the demo application is available, please [register you
 
 ### Install
 
-1.  Clone [the `blockprotocol` repository](https://github.com/blockprotocol/blockprotocol)
-1.  Run `yarn create-block-app yourblockname /path/to/yourblockname`
-1.  Switch to your new folder: `cd /path/to/yourblockname`
+1.  Move to a folder where you want to create your block
+1.  Run `npx create-block-app [your-block-name]` or `yarn create block-app [your-block-name]`
```

Comment:
```suggestion
1.  Run `npx create-block-app [your-block-name]` or `yarn create-block-app [your-block-name]`
```

Reply:
It should be `yarn create block-app` (with space). https://classic.yarnpkg.com/en/docs/cli/create/